### DISCUSSION TITLE
Possible to not use `ethers` even indirectly

### DISCUSSION THREAD

**Discussion by o-az** [wevm/wagmi#2457] [Q&A]
In my next app I switched from `ethers` to `viem` and `wagmi` latest version. I have no other web3 deps besides those & Rainbow. I run `yarn why ethers`:

<img width="1016" alt="image" src="https://github.com/wagmi-dev/wagmi/assets/23618431/ceeade65-42db-4cb0-b0cd-e5f2f980a0f0">

Shows me that a `wagmi` dep uses ethers: https://github.com/safe-global/safe-apps-sdk

To get more insight I use https://github.com/vercel/next.js#readme:

<img width="642" alt="image" src="https://github.com/wagmi-dev/wagmi/assets/23618431/65c0efaa-9093-4068-af4c-a26b4410e82c">

<img width="282" alt="image" src="https://github.com/wagmi-dev/wagmi/assets/23618431/f5bf217b-2dd4-49bf-a31c-f644da7db358">

I don't use any of these (`@safe-global/safe-apps-sdk`, `@coinbase/wallet-sdk`) and they're quite heavy. Is there a way to not include all this stuff or maybe I'm doing something wrong?

**Comment by tmm** [comment]
See https://github.com/wagmi-dev/wagmi/issues/2436

**Reply by o-az** [reply]
perfect. Thank you

**Reply by o-az** [reply]
What about coinbase wallet sdk. Is there a way to not include that?
It's not light

**Reply by o-az** [reply]
Maybe I should open another discussion? Discuss possibility of shipping a version that doesn't include all connectors by default. That would be fantastic
